lib/si-layer-cache/src/db.rs

```diff
@@ -17,28 +17,28 @@ use crate::{
 };
 
 use self::{
-    cache_updates::CacheUpdatesTask, cas::CasDb, function_execution::FunctionExecutionDb,
+    cache_updates::CacheUpdatesTask, cas::CasDb, func_execution::FuncExecutionDb,
     workspace_snapshot::WorkspaceSnapshotDb,
 };
 
 mod cache_updates;
 pub mod cas;
 pub mod encrypted_secret;
-pub mod function_execution;
+pub mod func_execution;
 pub mod serialize;
 pub mod workspace_snapshot;
 
 #[derive(Debug, Clone)]
-pub struct LayerDb<CasValue, EncryptedSecretValue, FunctionExecutionValue, WorkspaceSnapshotValue>
+pub struct LayerDb<CasValue, EncryptedSecretValue, FuncExecutionValue, WorkspaceSnapshotValue>
```

Comment:
Why do we need this to be a generic bound? if it only holds one FuncExectionValue, then it doesn't need to be another generic parameter.